code/game/objects/items/weapons/RCD.dm

```diff
@@ -72,7 +72,7 @@ RCD
 	set category = "Object"
 	set src in usr
 
-	if (!ishuman(usr) && !isrobot(usr))
+	if (!ishuman(usr) && !isrobot(usr) && !isdrone(usr))

```

Comment:
Why not use !usr.has_unlimited_silicon_privilege? That would let you eliminate the borg check as well.